src/api/forms/service.js

```diff
@@ -205,6 +209,8 @@ export async function createLiveFromDraft(formId, author) {
       `Live state not created from draft. Modified count ${result.modifiedCount}`
     )
   }
+
+  logger.info(`Live form created and draft form removed for form ID ${formId}`)
```

Comment:
We've already logged `Live form created… for form ID ${formId}` above?

```suggestion
  logger.info(`Draft form removed for form ID ${formId}`)
```